### PR TITLE
project template

### PR DESCRIPTION
**Pull request by yuusuf4real**
***Template Set up***

Two folders for both `css` and `js` including a file of `index.html` have been created with a welcoming message. kindly review

### CODE REVIEW

File: template/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
```
Comment (abdulfataiaka) [inline template/index.html]: Let's have a page title

File: template/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+</head>
+<body>
+
+	<header style="font-size: 54px; color: blue; text-align: center; font-family: cursive;">SMART LAUNDRY</header>
+
+	<p style="font-size: 24px; text-align: center;">....doing laundry the smartest way</p>
```
Comment (abdulfataiaka) [inline template/index.html]: We should stay away from using inline styles

**Comment by abdulfataiaka** [comment]
Create a file `.keep` in the js and css folders, Git does not track empty folders, thats why we don't see the css and js folders.

File: template/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+</body>
+</html>
```
Comment (abdulfataiaka) [inline template/index.html]: Always add an empty line at the end of every file

File: template/index.html (hunk 1 above)
Comment (yuusuf4real) [reply template/index.html]: Page title as been provided as `Smart Laundry`

File: template/index.html (hunk 2 above)
Comment (yuusuf4real) [reply template/index.html]: Inline css has been changed to external css

**Pull request merged**